scripts/firedrake-install

```diff
@@ -553,7 +553,8 @@ petsc_options = {"--with-fortran-bindings=0",
                  # Parallel mesh partitioner
                  "--download-ptscotch",
                  # For superlu_dist amongst others.
-                 "--with-cxx-dialect=C++11"}
+                 "--with-cxx-dialect=C++11",
+                 "--with-x=0"}
 
```

Comment:
Could we add this as a Mac only flag?
```python
if osname == "Darwin":
    petsc_options.add("--with-x=0")
```
The install doesn't currently fail on Linux

Reply:
Ah, ok. Done.